Sources/App/Core/Search.swift

```diff
@@ -164,8 +166,20 @@ enum Search {
             .column(license)
             .column(lastCommitDate)
             .column(lastActivityAt)
+            .column(dependenciesCount)
             .from(searchView)
             .from(SQLFunction("CONCAT", args: keywords), as: keyword)
+            .from(
+                SQLFunction(
+                    "COALESCE",
+                    args: SQLFunction(
+                        "array_length",
+                        args: resolvedDependencies, SQLLiteral.numeric("1")
+                    ),
+                    SQLLiteral.numeric("0") // default to zero if NULL or empty array
+                ),
+                as: dependenciesCount
+            )
```

Comment:
We have a `coalesce` function in `SQLKit+functions.swift`:

```swift
func coalesce(_ args: SQLExpression...) -> SQLFunction {
    SQLFunction("COALESCE", args: args)
}
```

which should work here. (There's also a `concat` in that file, which could perhaps replace that usage on line 171, which I missed in an earlier PR.)

Reply:
Will clean this up, thanks for pointing out the helper!